packages/snaps-execution-environments/package.json

```diff
@@ -41,8 +41,9 @@
   "scripts": {
     "auto-changelog-init": "auto-changelog init",
     "build": "ts-bridge --project tsconfig.build.json --verbose --clean --no-references",
-    "build:lavamoat": "lavamoat scripts/build.js --policy lavamoat/build-system/policy.json  --policyOverride lavamoat/build-system/policy-override.json",
-    "build:lavamoat:policy": "yarn build:lavamoat --writeAutoPolicy && node scripts/build.js --writeAutoPolicy",
+    "build:lavamoat": "lavamoat --policy lavamoat/build-system/policy.json  --override lavamoat/build-system/policy-override.json ./scripts/build.js",
+    "build:lavamoat:policy": "yarn build:lavamoat --writeAutoPolicy && LAVAMOAT_GENERATE_POLICY=true node scripts/build.js",
+    "build:lavamoat:test": "NODE_ENV=test yarn build:lavamoat",
```

Comment:
Not entirely sure how to feel about this, seems somewhat problematic that we aren't actually testing with prod builds in CI?

Reply:
The only difference is using `tsconfig` paths for testing, but not for production.

Reply:
Hmm, yeah I understand. Maybe worth the tradeoff